### PR TITLE
Weird behavious of Boolean expression evaluation of powershell caused…

### PR DESCRIPTION
**Pull request by nigurr**
Powershell doesn't evaluate expression before passing to command-let

### CODE REVIEW

File: Tasks/Gradle/Gradle.ps1 [hunk 1]
```diff
@@ -109,7 +109,7 @@ $singlemodule = [string]::IsNullOrEmpty($subprojects) -or $subprojects -eq '[]'
 if($isCoverageEnabled)
 {
    # Enable code coverage in build file
-   Enable-CodeCoverage -BuildTool 'Gradle' -BuildFile $buildFile -CodeCoverageTool $codeCoverageTool -ClassFilter $classFilter -ClassFilesDirectories $classFilesDirectories -SummaryFile $summaryFileName -ReportDirectory $reportDirectoryName -IsMultiModule !$singlemodule -ErrorAction Stop
+   Enable-CodeCoverage -BuildTool 'Gradle' -BuildFile $buildFile -CodeCoverageTool $codeCoverageTool -ClassFilter $classFilter -ClassFilesDirectories $classFilesDirectories -SummaryFile $summaryFileName -ReportDirectory $reportDirectoryName -IsMultiModule (!$singlemodule) -ErrorAction Stop
```
Comment (AshwiniChalla) [inline Tasks/Gradle/Gradle.ps1]: Version bump up required.

Comment (AshwiniChalla) [inline Tasks/Gradle/Gradle.ps1]: Add a E2E test for multi module Jacoco with this checkin

**Comment by AshwiniChalla** [comment]
signed off with comments

File: Tasks/Gradle/Gradle.ps1 (hunk 1 above)
Comment (acesiddhu) [inline Tasks/Gradle/Gradle.ps1]: do we need this for M91 as well? if yes we need to get approval

**Pull request merged**